VirtoCommerce.Storefront.Model/Lists/Services/IWishlistService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using PagedList.Core;
+using VirtoCommerce.Storefront.Model.Lists;
+
+namespace VirtoCommerce.Storefront.Model.Lists.Services
+{
+    public interface IWishlistService
+    {
+        Task<IPagedList<Wishlist>> SearchShoppingCartsAsync(WishlistSearchCriteria criteria);
```

Comment:
Need  rename to SearchWishListsAsync

Reply:
@tatarincev Don't use wish- prefix please, it's lists, not wishlists only

Reply:
since the main object is called "Wishlist" already shouldn't all utility services have this prefix?

Reply:
@ksavosteev it's not about specific place. yes, if we call it wishlists, then we should use it everywhere and you may ignore my comment. but I think it's not about wishlists only

Reply:
@ksavosteev @asvishnyakov I agree, it should be just lists, not specific wish lists